pkg/cli/cvd.go

```diff
@@ -71,13 +73,16 @@ func newCVDCommand(opts *subCommandOpts) *cobra.Command {
 		},
 	}
 	create.Flags().StringVar(&createFlags.Host, hostFlag, "", "Specifies the host")
+	create.Flags().StringVar(&createFlags.Branch, branchFlag, "aosp-master", "The branch name")
 	create.Flags().StringVar(&createFlags.BuildID, buildIDFlag, "", "Android build identifier")
 	create.Flags().StringVar(&createFlags.Target, targetFlag, "aosp_cf_x86_64_phone-userdebug",
 		"Android build target")
 	create.Flags().BoolVar(&createFlags.LocalImage, localImageFlag, false,
 		"Builds a CVD with image files built locally, the required files are https://cs.android.com/android/platform/superproject/+/master:device/google/cuttlefish/required_images and cvd-host-packages.tar.gz")
-	create.MarkFlagsMutuallyExclusive(buildIDFlag, localImageFlag)
-	create.MarkFlagsMutuallyExclusive(targetFlag, localImageFlag)
+	localImgMutuallyExFlags := []string{branchFlag, buildIDFlag, targetFlag}
+	for _, f := range localImgMutuallyExFlags {
+		create.MarkFlagsMutuallyExclusive(f, localImageFlag)
+	}
```

Comment:
Shouldn't `branchFlag` and `buildIDFlag` be marked as mutually exclusive too?
If you specify a build id the branch is implied, allowing it just opens a possible error where the build id doesn't match the branch.

Reply:
this is something I thought of but was not sure, done